packages/documentation-site/patternfly-docs/content/design-guidelines/components/switch/switch.md

```diff
@@ -3,80 +3,83 @@ id: Switch
+1. **Switch checkmark:** A checkmark, which can be used to further signify that a switch is turned on. When you use a switch label, a checkmark is optional. If you do not use a switch label, then a checkmark is required, to help users understand the status of a switch.
 
-1. **Form label:** Indicates what the switch turns on or off
-2. **Switch label:** Indicates whether the switch itself is turned on or off
-3. **Switch without label:** Indicates the state of a switch with a checkmark inside the switch itself, used in place of a switch label
-4. **Switch with label and checkmark:** Indicates the state of a switch with a checkmark inside the switch itself and label with additional message 
+## Usage 
 
-All switch types display their state (on or off) through different methods and locations. A basic switch communicates its state with an exterior switch label added to its right, while a switch without a label uses a checkmark to communicate its state from inside the switch itself.
+Use a switch to allow users to toggle a setting on and off. When a user toggles a switch, the change should save automatically without the need for additional action (like clicking a "Save" button).
 
-**General guidelines**
+Switches are most often used in forms, as a way of allowing users to turn on settings as they desire. The form label should specify what setting a switch turns on or off, such as "Bluetooth" or "Editing permissions":
 
-* Use a basic switch by default unless space is limited, then use a [switch without a label](/components/switch#without-label).
-* Add a form label in front of your switch to specify what setting a switch turns on or off, such as *Wi-fi* or *Bluetooth*.
-* If you're using a basic switch, add switch labels to clarify between toggled settings, such as *On* and *Off*.
-* Don't include the state of a switch in a label if you're using a switch with a label and a checkmark. 
-* Don’t use a switch if the options you’re presenting to the user are anything other than “on” or “off.” Instead, use [radio buttons](/components/forms/radio/design-guidelines). 
+![A switch that is used to set an option in a form](./img/switch-form.png)
 
+### When to use a switch vs. a radio button 
 
+If the available options represent anything other than “on” or “off”, use a [radio button](/components/forms/radio/design-guidelines) instead of a switch. 
 
-### When to use switches versus checkboxes
+### When to use a switch vs. a checkbox
 
-A switch changes an option and saves it simultaneously, while checkboxes require a separate action to save the selection, such as pressing a “Submit” or “Save” button.
+Use checkboxes when additional action is required to save a user's selections. A switch changes and saves an option simultaneously, while checkboxes do not. For example, the following image requires users to click the **Save** button:
```

Comment:
Would it be better to convey this as a recommendation, e.g. "A switch should change and save an option simultaneously, while checkboxes should not"?

Not a blocker, just something I noticed when checking out the link/image alt texts on the page